backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/migration/MigrationTaskSerializationTest.java

```diff
@@ -36,7 +36,7 @@
 
     private static final int SCHEMA_VERSION = 12;
     private static final String SERIALIZED_TASK = "{\"type\": \"CassandraMigration\", \"targetVersion\": 12}";
-    private static final String SERIALIZED_ADDITIONAL_INFORMATION = "{\"targetVersion\": 12}";
+    private static final String SERIALIZED_ADDITIONAL_INFORMATION = "{\"type\": \"CassandraMigration\", \"targetVersion\": 12}";
```

Comment:
as far as I know we decided to use kebab case for serialized types, could you fix that for all AdditionalInformation?
It could very well be done in another PR.

Reply:
please open a specific ticket for normalizing the types please

Reply:
https://ci.linagora.com/linagora/lgs/openpaas/james/issues/1481